### PR TITLE
Make symbol loading more than 50x faster on Linux

### PR DESCRIPTION
**Pull request by pierricgimmig**
Remove N^2 redundant sorting of functions data views when loading symbols. 
On Linux, this makes Unreal symbols load in ~5 seconds instead of more than 
4 minutes. On Windows the gains are a bit less because we are limited by the poor 
performance of the DiaSdk for symbol loading. An upcoming PR will introduce 
raw_pdb for another massive improvement on Windows.

When loading symbols, we would call "OnDataChanged" and "DoSort" for every 
symbol added to DataViews: 

orbit_symbols_4_minutes

Don't call OnDataChanged() on individual add/remove operations in FunctionsDataView to 
allow a single batched sort of all previously added/removed elements. We also introduce a 
mechanism to disable refresh callbacks so that batched operations like the initial loading of 
all debug symbols don't trigger an absurd number of refreshes.

### CODE REVIEW

**Review by florian-kuebler — APPROVED** [review]
LGTM. But please also wait for @beckerhe for second approval.

**Review by beckerhe — APPROVED** [review]
I'm sorry Pierric. I completely missed this. LGTM!

(PS: Great to hear from you - even if it's just digitally)

**Comment by pierricgimmig** [comment]
Thanks @beckerhe , thanks @florian-kuebler !

**Pull request merged**
